Demo enhancement (#91)
ml4sts/benchtools: docs/source/demo.md

First, clone the repo: 

> ```{code-block} bash

Reviewer: no. I used code-cell on purpose for rendering style

Author: What should be done for the cells to render? They weren't rendering on my end when I was reviewing the docs

Reviewer: oh, if they't not rendering then maybe the files need different yaml header to render right. `code-cell` renders in md notebooks

We can run a benchmark by name

::::{note}

> The default runner is ollama. Make sure ollama is running by calling `ollama serve` on another terminal window

Reviewer: ```suggestion
The default runner is `ollama`. Make sure ollama is running in advance. 
```

I'm pretty sure I have never run `ollama serve` so I don't think that was completely accurate. I'd prefer to add a link here to ollama docs for how to make sure it's running. (across platforms)

```{code-block} bash
:tags: ["skip-execution"]
benchtool run .
```

Let's see what happened after running the benchmark

```{code-block} bash
:tags: ["skip-execution"]
ls
about.md	info.yml	logs tasks.yml

```

Now we have a new `logs/` folder. Let's explore its contents 
```{code-block} bash
:tags: ["skip-execution"]
ls logs/
gemma3

```

```{code-block} bash
:tags: ["skip-execution"]
ls logs/gemma3/
animal

```

```{code-block} bash
:tags: ["skip-execution"]
ls logs/gemma3/animal
1780063281

```
`1780063281` is the timestamp of when the benchmark was run which represents a single run of the benchmark

```{code-block} bash
:tags: ["skip-execution"]
ls logs/gemma3/animal/1780063281

> animal_spots-hairy-paws  animal_stripes-hairy-hooves  animal_stripes-hairy-paws  run_info.yml

Reviewer: dont'e mix input and output

```
Each run will have a folder for the different subtasks  

Let's run the scorer to score the LLM on the task at hand.

```{code-block} bash

> :tags: ["skip-execution"]

Reviewer: this is only for code-cells; so remove if using code-block on purpose

benchtool score .
```

> ```{code-block} consol

Reviewer: ```suggestion
```{code-block} console
```

```{code-block} bash
:tags: ["skip-execution"]
benchtool run .
```

> ```{code-block} bash

Reviewer: why are you adding steps here without explanation?
